tests/integration/multicluster/centralremotekubeconfig/main_test.go

```diff
@@ -32,13 +32,13 @@ var ist istio.Instance
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
-		Skip("https://github.com/istio/istio/pull/33045").
+		// Skip("https://github.com/istio/istio/pull/33045").
 		Label(label.Multicluster).
 		RequireMinClusters(2).
 		Setup(func(ctx resource.Context) error {
 			// TODO, this should be exclusively configurable outside of the framework
-			configCluster := ctx.Clusters()[0]
-			externalControlPlaneCluster := ctx.Clusters()[1]
+			configCluster := ctx.Clusters()[1]
```

Comment:
using index seems flaky. I think `ctx.Clusters().{Remotes,Primaries}()` can do this better?

Reply:
if not - we should make functions that do it better :slightly_smiling_face:

Reply:
I'm not sure this test should stay around much longer anyway - it doesn't cover what we do for most multicluster tests (pod to pod traffic, telemetry, etc)

This is all configurable at the framework level, I figured we would add new jobs that set this in the topology file.

Reply:
Yes, this could be updated, but I basically left it as it was, changing what was installed on each to match the istio.io documentation. The documentation has the user install the config cluster on the `remote` cluster, and the control plane on the `external` cluster (used the `primary` cluster since that is what we have).